### PR TITLE
Updated v2 context

### PR DESCRIPTION
**Pull request by TelegramSam**
Signed-off-by: Sam Curren <telegramsam@gmail.com>

### CODE REVIEW

File: site/static/messaging/contexts/v2/index.json [hunk 1]
```diff
@@ -3,7 +3,18 @@
...
        "DIDCommMessaging":"https://didcomm.org/messaging/v2/#DIDCommMessaging",
-       "accept":"https://didcomm.org/messaging/v2/#accept",
-       "routingKeys":"https://didcomm.org/messaging/v2/#routingKeys"
+	   "serviceEndpoint": {
+          "@id": "https://www.w3.org/ns/did#serviceEndpoint",
+          "@type": "@id"
+		  "@container": "@list"
+		  "@context": {
+			"uri": {
+				"@container": "@list",
```
Comment (TelegramSam) [inline site/static/messaging/contexts/v2/index.json:12]: not a list, single URI.

**Comment by TelegramSam** [comment]
need review by @OR13

**Comment by OR13** [comment]
https://v.jsld.org and some testing can resolve this.

**Comment by OR13** [comment]
@TelegramSam  does this look correct? 

https://v.jsld.org/a5yy9gGBiHRPaBJY33DgjmSwAtH4JpJaVq6raUW2qaANVWcPqzF4QcrnMaL7b5S7Efhn31N9hgYPWKZasF6Eeswkp5xs9i5PbLGdYeaUFf2Tr7vZ1jXBTzvHDuEyC33vv1sVJME3frxhfY2epKAxVtnV5PhLzWW9Cg8uXSLXU89uDm2qdrFYnCACnk3bk4kXpMZ2fV8jspd4VDb3DLDxmEPqFcpWcemWyrxNXkXAaJoPGphFeRaouSSyhPbkrX1cfSRMiQaW4coioUWrAL57VoqPwXjtsjK5dynFeYyKY9rtiMdLYFEknP9PgCVrFpvy5qzfuG54CF5eMDpgo2E4mYcvnLdWSTMBSEVa7nuaHuyEn8ha3x623q6TZbdtMcg5eSTCr4VkoJzFj2HT6NWecuQtGQeFhADaNJT6mCHBBumkTJiAh2ezCw6aJf2GzcW2c8ETSkgRfZnRgy5yduYEDzdsTLsZ2Z933ME5KPhZiVuaqbLaUgGpxbxpyrav3QK4NKTsztyTFVP

**Comment by OR13** [comment]
your edits are not parseable json.... You can use this to clean them up:

https://v.jsld.org/a5yy9gGBiHRPaBJY33DgjmSwAtH4JpJaVq6raUW2qaANVWcPqzF4QcrnMaL7b5S7Efhn31N9hgYPWKZasF6Eeswkp5xs9i5PbLGdYeaUFf2Tr7vZ1jXBTzvHDuEyC33vv1sVJME3frxhfY2epKAxVtnV5PhLzWW9Cg8uXSLXU89uDm2qdrFYnCACnk3bk4kXpMZ2fV8jspd4VDb3DLDxmEPqFcpWcemWyrxNXkXAaJoPGphFeRaouSSyhPbkrX1cfSRMiQaW4coioUWrAL57VoqPwXjtsjK5dynFeYyKY9rtiMdLYFEknP9PgCVrFpvy5qzfuG54CF5eMDpgo2E4mYcvnLdWSTMBSEVa7nuaHuyEn8ha3x623q6TZbdtMcg5eSTCr4VkoJzFj2HT6NWecuQtGQeFhADaNJT6mCHBBumkTJiAh2ezCw6aJf2GzcW2c8ETSkgRfZnRgy5yduYEDzdsTLsZ2Z933ME5KPhZiVuaqbLaUgGpxbxpyrav3QK4NKTsztyTFVP

**Comment by brianorwhatever** [comment]
I took another stab at it but am still just mostly guessing. The tools don't like when I try to redefine the protected "serviceEndpoint" term. This is my latest however Orie's tool explodes on a "termType" error

https://v.jsld.org/a5yy9gGBiHRPaBJY33DgjmSwAtH4JpJaVq6raUW2qaANVWcPqzF4QcrnMaL7b5S7Efhn31N9hgYPWKZasF6Eeswkp5xs9i5PbLGdYeaUFf2Tr7vZ1jXBTzvHDuEyC33vv1sVJME3frxhfY2epKAxVtnV5PhLzWW9Cg8uXSLXU89uDm2qdrFYnCACnk3bk4kXpMZ2fV8jspd4VDb3DLDxmEPqFcpWcemWyrxNXkXAaJoPGphFeRaouSSyhPbkrX1cfSRMiQaW4coioUWrAL57VoqPwXjtsjK5dynFeYyKY9rtiMdLYFEknP9PgCVrFpvy5qzfuG54CF5eMDpgo2E4mYcvnLdWSTMBSEVa7nuaHuyEn8ha3x623q6TZbdtMcg5eSTCr4VkoJzFj2HT6NWecuQtGQeFhADaNJT6mCHBBumkTJiAh2ezCw6aJf2GzcW2c8ETSkgRfZnRgy5yduYEDzdsTLsZ2Z933ME5KPhZiVuaqbLaUgGpxbxpyrav3QK4NKTsztyTFVP

**Comment by brianorwhatever** [comment]
I think we are having issues trying to turn this context file into validation of the schema as normatively defined in the spec. After looking over a number of context files they rarely do things like add "@list" containers. I propose we keep this context simple with just semantics and leave spec validation to implementing libraries. Here is a simple didcomm service schema:

```json
{
      "@version": 1.1,
      "id": "@id",
      "type": "@type",
      "DIDCommMessaging": "https://didcomm.org/messaging/v2#DIDCommMessaging",
      "uri": "https://didcomm.org/messaging/v2#uri",
      "accept": "https://didcomm.org/messaging/v2#accept",
      "routingKeys":  "https://didcomm.org/messaging/v2#routingKeys"                
}
```

File: site/static/messaging/contexts/v2/index.json [hunk 2]
```diff
@@ -3,7 +3,18 @@
        "@version":1.1,
        "@protected":true,
        "DIDCommMessaging":"https://didcomm.org/messaging/v2/#DIDCommMessaging",
-       "accept":"https://didcomm.org/messaging/v2/#accept",
-       "routingKeys":"https://didcomm.org/messaging/v2/#routingKeys"
+	   "serviceEndpoint": {
```
Comment (vongohren) [inline site/static/messaging/contexts/v2/index.json:6]: Is the v2 spec getting a service endpoint attribute afterall?

**Comment by dhh1128** [comment]
Note: reviewed (positively) by DIDComm WG on 2022-05-30. WG votes "merge."

**Pull request merged**